fitwsarah_frontend/src/config.json

```diff
@@ -0,0 +1,6 @@
+{
+    "domain": "dev-twa7h1nv0usycyum.us.auth0.com",
```

Comment:
If you don't mind just explaining what these 4 things do I don't really know much bout authentication and I'm just curious?

Reply:
This is the domain of auth0 to connect to it.